brainreg/backend/niftyreg/niftyreg_binaries.py

```diff
@@ -9,17 +11,36 @@
     os_folder_name = __os_folder_names[platform.system()]
 except KeyError:
     raise ValueError(
-        "Platform {} is not recognised as a valid platform. "
-        "Valid platforms are : {}".format(
-            platform.system(), __os_folder_names.keys()
-        )
+        f"Platform {platform.system()} is not recognised as a valid platform. "
+        f"Valid platforms are : {__os_folder_names.keys()}"
     )
 
 
-def get_binary(binaries_folder, program_name):
-    path = os.path.join(binaries_folder, os_folder_name, program_name)
-    return path
+def conda_install_path() -> Optional[Path]:
+    """
+    If a conda install of niftyreg is available, return the directory
+    containing the niftyreg binaries.
+    """
+    if "CONDA_PREFIX" not in os.environ:
+        return
```

Comment:
```suggestion
        return None
```
(detail) Worth being more explicit? Or is it common practice not to? (Not sure/I didn't know)

Reply:
I think it's pretty common practice to just have a bare return, and there might even be style checkers that prefer this.

Reply:
Ah it's actually the other way round, and `pre-commit` enforces `return None` - I guess this is clearer, thanks for catching 😄